meilisearch-release-changelog/v1.3.0.md

```diff
@@ -0,0 +1,305 @@
+
+Done by @Kerollmops in #3825
+
+
+## Display ranking scores at search
+
+Meilisearch v1.3.0 introduces ranking scores per document via the `showRankingScore` search query parameter. 
+
+```bash 
+curl \
+  -X POST 'http://localhost:7700/indexes/movies/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "Batman Returns", "showRankingScore": true }'
+```
+
+Using this search query parameter means that each document will return with a `_rankingScore` property displaying a score between 0 and 1. The higher the ranking score, the more relevant the document.
+
+```json 
+"_rankingScore": 0.8575757575757575,
+```
+
+### Ranking score details - Experimental 
+
+Additionally, as an experimental feature, you can view detailed scores per ranking rule for each document via the `showRankingScoreDetails` search query parameter. 
+
+```bash 
+curl \
+  -X POST 'http://localhost:7700/indexes/movies/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "Batman Returns", "showRankingScoreDetails": true }'
+```
+
+With this search query parameter in place each document will return with an `_rankingScoreDetails` JSON object. That object will contain a property and a score value for each ranking rule. 
+
+```json 
+"_rankingScoreDetails": {
+  "words": {
+    "order": 0,
+    "matchingWords": 1,
+    "maxMatchingWords": 1,
+    "score": 1.0
+  },
+  "typo": {
+    "order": 1,
+    "typoCount": 0,
+    "maxTypoCount": 1,
+    "score": 1.0
+  },
+  "proximity": {
+    "order": 2,
+    "score": 1.0
+  },
+  "attribute": {
+    "order": 3,
+    "attributes_ranking_order": 0.8,
+    "attributes_query_word_order": 0.6363636363636364,
+    "score": 0.7272727272727273
+  },
+  "exactness": {
+    "order": 4,
+    "matchType": "noExactMatch",
+    "score": 0.3333333333333333
+  }
+}
+```
+
+This experimental feature can be turned on via the HTTP API using v1.3.0's new `/experimental-features` endpoint. 
+
+```bash
+ curl \
+  -X PATCH 'http://localhost:7700/experimental-features/' \
+  -H 'Content-Type: application/json'  \
+  --data-binary '{
+    "scoreDetails": true
+  }'
+
+```
+
+🗣️ Since this feature is experimental, we need your help to iterate on it! You are more than welcome to share your thoughts and your feedback on [this discussion](https://github.com/meilisearch/product/discussions/674).
+
+⚠️ The experimental feature API can break between Meilisearch versions. Be aware of this if using it in production.
+
+### Relevancy change on `attribute` ranking rule
+
+Previously, the documents with attributes containing the query words at the beginning of the attribute would be considered more relevant than documents containing the query words at the end of the attributes.
+
+With this change the ranking rule considers the distance to the position of the word in the query rather than the absolute distance to the beginning of the document.
```

Comment:
Most of the times it's better to begin with the change, as many people will be skimming the changelog and only read the first few words of a section.

```suggestion
The `attribute` ranking rule now determines relevancy based on the distance to the position of the word in the query rather than the absolute distance to the beginning of a document.

Previously, documents with attributes containing search terms at the beginning of the attribute would be considered more relevant than documents containing search terms at the end of an attribute.
```